projects/igniteui-angular/schematics/ng-add/schema.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "id": "igniteui-angular-ng-add",
+    "title": "IgniteUI for Angular ng-add schematic",
+    "type": "object",
+    "properties": {
+        "polyfills": {
+            "type": "boolean",
+            "description": "Add polyfills for IE.",
+            "default": "false",
+            "x-prompt": "Would you like to add polyfills for IE?"
```

Comment:
those are already present in the project, correct? So more like "enable" rather than "add".